server/main.py

```diff
@@ -37,24 +37,26 @@ async def lifespan(app: FastAPI):
 
     # static is the default folder that fastapi uses for grabbing
     # javascript and css files
-    web_path_for_css_and_javascript_files = '/static'
-    app.mount(
-        web_path_for_css_and_javascript_files, 
-        StaticFiles(directory='./server/frontend/static'), 
-        name='static'
-    )
-    
+
+    # web_path_for_css_and_javascript_files = '/static'
+    # app.mount(
+    #     web_path_for_css_and_javascript_files,
+    #     StaticFiles(directory='./server/frontend/static'),
+    #     name='static'
+    # )
```

Comment:
Should be uncommited because we are coding in the prespective of the user, not the developer